tests/scripts/test_PMID_downloader.py

```diff
@@ -35,111 +40,115 @@ def pdf_bytes():
 
 
 @pytest.fixture()
-def pmids():
+def pmids_set():
```

Comment:
If you want to make clear that this function returns/is a set. Pleas do:
```suggestion
def pmids() -> set:
```